src/api/models/Activity.ts

```diff
@@ -95,9 +103,18 @@ Activity.init(
                 }
             }
         },
+        costPerIndividual: {
+            type: DataTypes.FLOAT,
+            defaultValue: 0.0
+        },
+        costPerGroup: {
+            type: DataTypes.FLOAT,
+            defaultValue: 0.0
+        },
+        image: DataTypes.STRING,
         startTime: {
             type: DataTypes.DATE,
-            defaultValue: new Date(Date.now()).toISOString()
+            defaultValue: new Date(0).toISOString()
```

Comment:
Why is the 0 better than Data.Now()? I think it makes more sense that the start time be when it was created.

Reply:
Date(0) is easier for the clilent to realize they forgot to put a startDate, then they can update it later.